generator/graphql-kotlin-federation/README.md

```diff
@@ -4,7 +4,7 @@
 
 `graphql-kotlin-federation` extends the functionality of `graphql-kotlin-schema-generator` and allows you to easily
 generate federated GraphQL schemas directly from the code. Federated schemas rely on a number of directives to
-instrument the behavior of the underlying graph, see [the documentation](https://expediagroup.github.io/graphql-kotlin/docs/federated/federated-directives).
+instrument the behavior of the underlying graph, see [the documentation](https://expediagroup.github.io/graphql-kotlin).
```

Comment:
broken link?

Reply:
This is in the README. Instead of always having to update we can just reference the docs in general